google/cloud/spanner/client_test.cc

```diff
@@ -395,12 +397,19 @@ TEST(ClientTest, MakeConnectionOptionalArguments) {
   EXPECT_EQ(conn->options().get<EndpointOption>(),
             spanner_internal::DefaultOptions().get<EndpointOption>());
 
-  conn = MakeConnection(db, Options{}.set<EndpointOption>("endpoint"));
-  ASSERT_NE(conn, nullptr);
-  ASSERT_TRUE(conn->options().has<EndpointOption>());
-  EXPECT_NE(conn->options().get<EndpointOption>(),
-            spanner_internal::DefaultOptions().get<EndpointOption>());
-  EXPECT_EQ(conn->options().get<EndpointOption>(), "endpoint");
+  {
```

Comment:
nit: remove the braces?  There is no code after you close the explicit scope.

Reply:
My thought: One day there might be extra code, or something might get moved, and in the meantime it documents the desired extent of the effect.